DFC.ServiceTaxonomy.Editor/wwwroot/ncs.js

```diff
@@ -0,0 +1,10 @@
+﻿$(function () {
+    var ul = $("ul#adminMenu li#new ul");
+    var li = ul.children("li");
+
+    li.detach().sort(function (a, b) {
+        return (a.innerText > b.innerText) - (a.innerText < b.innerText);
```

Comment:
This looks a bit odd to me, with the boolean arithmetic.
Perhaps an easier to follow bit of code would be better, even it it takes a few more lines?

Reply:
Done!